java/core/src/java/org/apache/orc/impl/reader/StripePlanner.java

```diff
@@ -279,12 +280,13 @@ private void buildEncodings(OrcProto.StripeFooter footer,
   private long handleStream(long offset,
                             boolean[] columnInclude,
                             OrcProto.Stream stream,
+                            StreamName.Area area,
```

Comment:
Could you add `area` into the function description, too?

Reply:
Added in cb829a1.  But the javadoc generation failed. It seems to be caused by network instability.